src/base/InternalBaseBlock.php

```diff
@@ -109,7 +112,7 @@ public function offsetGet($offset)
      */
     public function getIsCacheEnabled()
     {
-        return $this->cacheEnabled;
+        return $this->isCachable();
```

Comment:
This is not true, as it should return whether caching is available for this block or not and not whether caching component exists, so i think you can revert those changes here, but everything is perfect in PhpBlock. Afterwards we can merge. Yes! :-)